code-challenges.js

```diff
@@ -18,8 +34,19 @@ var cubeAndSum1 = [2, 3, 4]
 var cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
 
+//make function called 'cubed' that takes in an array and cube every index in that array and find the summation of the indexs after being cubed. For loop to iterate through every number
+
+const cubed = (arr) => {
+  //for(let i=0; i < arr.length; i++){
+    arr = (arr[0] * arr[0] * arr[0]) + (arr[1] * arr[1] * arr[1]) + (arr[2] * arr[2] * arr[2])
```

Comment:
so this solution works for the test variables we provided. but how could your function handle arrays that are not 3 elements long? how can we use some sort of iteration to make this dynamic?